scripts/run_performance_job.py

```diff
@@ -0,0 +1,518 @@
+from dataclasses import dataclass, field
```

Comment:
Not sure exactly where to put this comment, but do we actually support running against WASM as this currently stands? I am having a hard time understanding where we package up the various runtime packs and such. Is that something that just needs to be handled by the user?

Reply:
no it doesn't support wasm right now, I have a TODO for it in the code but I need to understand more about what all the inputs are and where it downloads everything

Reply:
I know there are some files in the runtime repo that are used in wasm runs, and so to support that we would need to add additional sources to the helix.yml which ensures those directories are sent to crank